nvm.sh

```diff
@@ -1907,6 +1908,57 @@ nvm_download_artifact() {
   nvm_echo "${TARBALL}"
 }
 
+# args: tarball path, NVM_OS var
+nvm_extract_tar_artifact() {
+  local tar_compression_flag
+  local TARBALL
+  local TAR_COMMAND_SUFFIX
+  local TMPDIR
+  local NVM_OS
+  local tar
+
+  TARBALL="${1}"
+  NVM_OS="${2}"
+
+  tar='tar'
+  if [ "${NVM_OS}" = 'aix' ]; then
+    tar='gtar'
+  fi
+
+  tar_compression_flag='z'
+
+  if nvm_supports_xz "${VERSION}" && [ "_$NVM_OS" != "_openbsd" ]; then
```

Comment:
lol, i mean, if every single possible version of `tar` will either error with the same error message, or support `-J`, then that's pretty robust

Reply:
I couldn't test all possible versions, but `GNU tar 1.28` on Linux and `bsdtar 2.8.3` on macosx all provide sufficiently clear error messages to understand whether the support for `J` option exists.

I'm adding a test to the specific error string given by `tar` on OpenBSD, that should solve the issue for OpenBSD. Since I didn't test on other operating systems (other than the ones mentioned above), I'm not sure it is enough to go through it without checking the OS. On the other hand this situation only happens on OpenBSD.

I'm uploading a new version.